contracts/TitleEscrow.sol

```diff
@@ -21,11 +22,11 @@ contract HasNamedBeneficiary is Context {
   }
 }
 
-contract HasHolder is Context {
-  address public holder;
+abstract contract HasHolder is Context, IHasHolder {
+  address public override holder;
 
-  event HolderChanged(address indexed previousHolder, address indexed newHolder);
-  constructor(address _holder) internal {
+  //   event HolderChanged(address indexed previousHolder, address indexed newHolder);
```

Comment:
do we still need this line?

Reply:
oops nope, it's been moved to the interface definition